src/services/search.service.ts

```diff
@@ -10,21 +10,26 @@ import { CipherType } from '../enums/cipherType';
 import { FieldType } from '../enums/fieldType';
 import { UriMatchType } from '../enums/uriMatchType';
 import { SendView } from '../models/view/sendView';
+import { I18nService } from '../abstractions';
 
 export class SearchService implements SearchServiceAbstraction {
     private indexing = false;
     private index: lunr.Index = null;
+    private searchMinEdge = 2;
 
-    constructor(private cipherService: CipherService, private logService: LogService) {
+    constructor(private cipherService: CipherService, private logService: LogService, private i18nService: I18nService) {
```

Comment:
Please wrap `private i18nService: I18nService` onto the next line, constructor arguments should be 2 per line when exceeding 2 arguments.